### PR TITLE
Added note above cross-domain redirects.

### PR DESCRIPTION
**Pull request by pamgoodrich**
Address #220 .

Added a note about cross-domain redirects.. Light style editing.

### CODE REVIEW

File: 4.-documentation/e2.configure-redirects.md [hunk 1]
```diff
@@ -4,24 +4,26 @@
...
-> This feature is available on the [Stoplight **Starter** plan](https://stoplight.io/pricing/), and above.
+> This feature is available on the [Stoplight **Starter** plan](https://stoplight.io/pricing/) and above.
 
-## How it works
+## How It Works
 
-Within your Stoplight Workspace, you can configure redirect rules and a custom domain. When someone navigates to a path matching one of your custom domain's redirect rules, they will automatically be redirected to the new path.
+Within your Stoplight workspace, you can configure redirect rules and a custom domain. When someone navigates to a path matching one of your custom domain's redirect rules, they will automatically be redirected to the new path.
+
+> Redirects across domains are not supported.
```
Comment (lottamus) [inline 4.-documentation/e2.configure-redirects.md:15]: Do we need to call it out this large? What if we just added it to the end of the paragraph above.

image

```suggestion
Within your Stoplight workspace, you can configure redirect rules and a custom domain. When someone navigates to a path matching one of your custom domain's redirect rules, they will automatically be redirected to the new path. Redirects across domains are not supported.
```

**Review by lottamus — APPROVED** [review]
Any idea why we don't support redirecting to another domain? It would be very simple to support.

File: 4.-documentation/e2.configure-redirects.md (hunk 1 above)
Comment (pamgoodrich) [reply 4.-documentation/e2.configure-redirects.md]: Good point. I rewrote the paragraph a bit:

Within your Stoplight workspace, you can configure a custom domain and redirect rules within the same domain. When someone navigates to a path matching one of your custom domain's redirect rules, they will automatically be redirected to the new path. 

Does that work?

**Comment by pamgoodrich** [comment]
@naumanali-stoplight : @lottamus asked why we don't support cross-domain redirects. Can you please let us know?

**Pull request merged**